sources/packages/web/src/services/http/dto/Announcements.dto.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Announcements details.
+ */
+export interface AnnouncementsAPIOutDTO {
+  message_title: string;
```

Comment:
`messageTitle`? Likewise for the start and end dates.

Reply:
updated